source/simulator/core.cc

```diff
@@ -1043,6 +1046,7 @@ namespace aspect
             {
               coupling[x.velocities[d]][x.pressure] = DoFTools::always;
               coupling[x.pressure][x.velocities[d]] = DoFTools::always;
+              coupling[x.pressure][x.pressure] = DoFTools::always;
```

Comment:
Or make a new signal that takes the same arguments as post_constraints_creation? I noticed that the pressure coupling IS already present in the preconditioner matrix, presumably because for a cheap preconditioner something like 1/eta is put on the diagonal of the pressure block?